mypy/checkmember.py

```diff
@@ -855,6 +855,8 @@ class B(A[str]): pass
                                                               builtin_type, original_type,
                                                               original_vars=original_vars))
                            for item in t.items()])
+    if isuper is not None:
+        t = cast(ProperType, expand_type_by_instance(t, isuper))
```

Comment:
It would be nice to have a comment here, or maybe add a case to the above comment where the type is not callable.

Also, I have trouble convincing myself how we can be sure that the type arguments from `isuper` can always be applied to `t`. Maybe explain this is briefly (in the above comment perhaps, since this doesn't seem specific to this case).

Reply:
> Also, I have trouble convincing myself how we can be sure that the type arguments from `isuper` can always be applied to `t`.

I actually have trouble understanding how this can _not_ work :-) My guess is confusion comes from missing docstring entry for `isuper`, which is current instance mapped to the method definition superclass. I was also thinking about an assert, but this would be probably too much.

Reply:
Btw, while adding docstring items I noticed there are two unused arguments in this function.

Reply:
Yeah, I was confused about `isuper`. Now it all makes sense.